examples/showcase/bachelors_degrees_by_gender.py

```diff
@@ -1,3 +1,11 @@
+"""Timeline graph comparison with aligned colored annotation.
+
+This example shows lines tracking change over time.  It features dashed
+lines at fixed increments on the Y axis.
```

Comment:
What about something like...
"""
A graph of multiple time series which demonstrates extensive custom
styling of plot frame, tick lines and labels, and line graph properties.

Also demonstrates the custom placement of text labels along the right edge
as an alternative to a conventional legend.
"""